airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/DatabricksConnectorClientsFactory.kt

```diff
@@ -44,18 +44,23 @@ object DatabricksConnectorClientsFactory {
         // EnableArrow=0 flag is undocumented and disables ArrowBuf when reading data
         // Destinations only reads data for metadata or for comparison of actual data in tests. so
         // we don't need it to be optimized.
-        val jdbcUrl =
-            "jdbc:databricks://${config.hostname}:${config.port}/${config.database};transportMode=http;httpPath=${config.httpPath};EnableArrow=0"
```

Comment:
@gisripa do you remember what the `/${config.database}` thing is supposed to do? It sounds like we're passing in the database (i.e. unity catalog name?)

but the oss driver's [docs](https://docs.gcp.databricks.com/en/integrations/jdbc/oss.html#configure-the-connection-url) sound like this is actually supposed to be the schema, which sounds super weird. I.e. their url format is `jdbc:databricks://<server-hostname>:<port>/<schema>;`

(I'm seeing some weird test failures with ConcurrentModificationException and am very confused about how that could be happening, no idea if this is related)